Investigate and fix failed checks from upward merge

|Wazuh version|Install type|Action performed|Platform|
|---|---|---|---|
| 4.8.0 | Manager | Install | - |

## Description

Checks have failed on the next merge upward PR for a variety of reasons, although all of these changes have been merged into `4.8.0` - `Beta 1`:
- https://github.com/wazuh/wazuh-packages/pull/2814

We need to check the root of the problem and fix it so that the checks pass correctly.

### Error details

- [test-password-tool-success](https://github.com/wazuh/wazuh-packages/actions/runs/7815955726/job/21362553210#logs)
```
  wazuh-passwords-tool.sh: line 340: /var/ossec/bin/wazuh-keystore: No such file or directory
  Unauthorized401
  Error: Process completed with exit code 1.
``` 

- [All In One - Amazonlinux 2_base - Tests](https://github.com/wazuh/wazuh-packages/pull/2814/checks?check_run_id=21321133221)
- [All In One - Rhel 9 - Tests](https://github.com/wazuh/wazuh-packages/pull/2814/checks?check_run_id=21321156851)
- [Distributed Cases - manager_dashboard - centos - Tests](https://github.com/wazuh/wazuh-packages/pull/2814/checks?check_run_id=21321781529)
- [Distributed Cases - manager_indexer - ubuntu - Tests](https://github.com/wazuh/wazuh-packages/pull/2814/checks?check_run_id=21321905341)
```
indexer-connnector: ERROR: HTTP response code said error, status code: 401
```

## DoD

- [ ] Find the cause of the problems.
- [ ] Fix the issues encountered.
- [ ] Re-run the checks to verify that they are working correctly.

--- a/.github/actions/passwords-tool/tests-stack-success.sh
+++ b/.github/actions/passwords-tool/tests-stack-success.sh
@@ -22,7 +22,7 @@ echo '::endgroup::'
 
 echo '::group:: Change all passwords except Wazuh API ones.'
 
-mapfile -t pass < <(bash wazuh-passwords-tool.sh -a | awk '{ print $NF }' | sed \$d | sed '1d' )
+mapfile -t pass < <(bash wazuh-passwords-tool.sh -a | grep 'The password for' | awk '{ print $NF }')
 for i in "${!users[@]}"; do
     if curl -s -XGET https://localhost:9200/ -u "${users[i]}":"${pass[i]}" -k -w %{http_code} | grep "401"; then
         exit 1
@@ -35,7 +35,7 @@ echo '::group:: Change all passwords.'
 
 wazuh_pass="$(cat wazuh-install-files/wazuh-passwords.txt | awk "/username: 'wazuh'/{getline;print;}" | awk '{ print $2 }' | tr -d \' )"
 
-mapfile -t passall < <(bash wazuh-passwords-tool.sh -a -au wazuh -ap "${wazuh_pass}" | awk '{ print $NF }' | sed \$d ) 
+mapfile -t passall < <(bash wazuh-passwords-tool.sh -a -au wazuh -ap "${wazuh_pass}" | grep 'The password for' | awk '{ print $NF }' ) 
 passindexer=("${passall[@]:0:6}")
 passapi=("${passall[@]:(-2)}")
 
@@ -63,7 +63,7 @@ echo '::endgroup::'
 
 echo '::group:: Change all passwords except Wazuh API ones using a file.'
 
-mapfile -t passfile < <(bash wazuh-passwords-tool.sh -f wazuh-install-files/wazuh-passwords.txt | awk '{ print $NF }' | sed \$d | sed '1d' )
+mapfile -t passfile < <(bash wazuh-passwords-tool.sh -f wazuh-install-files/wazuh-passwords.txt | grep 'The password for' | awk '{ print $NF }' ) 
 for i in "${!users[@]}"; do
     if curl -s -XGET https://localhost:9200/ -u "${users[i]}":"${passfile[i]}" -k -w %{http_code} | grep "401"; then
         exit 1
@@ -72,7 +72,7 @@ done
 echo '::endgroup::'
 
 echo '::group:: Change all passwords from a file.'
-mapfile -t passallf < <(bash wazuh-passwords-tool.sh -f wazuh-install-files/wazuh-passwords.txt -au wazuh -ap BkJt92r*ndzN.CkCYWn?d7i5Z7EaUt63 | awk '{ print $NF }' | sed \$d ) 
+mapfile -t passallf < <(bash wazuh-passwords-tool.sh -f wazuh-install-files/wazuh-passwords.txt -au wazuh -ap BkJt92r*ndzN.CkCYWn?d7i5Z7EaUt63 | grep 'The password for' | awk '{ print $NF }' )
 passindexerf=("${passallf[@]:0:6}")
 passapif=("${passallf[@]:(-2)}")
 

